make/hotspot/lib/CompileJvm.gmk

```diff
@@ -148,7 +148,7 @@ LIBJVM_STATIC_EXCLUDE_OBJS := operator_new.o
 ifeq ($(call isTargetOs, windows), true)
   WIN_EXPORT_FILE := $(JVM_OUTPUTDIR)/win-exports.def
 
-  JVM_LDFLAGS += -def:$(WIN_EXPORT_FILE)
+  JVM_LDFLAGS_NO_GTEST += -def:$(WIN_EXPORT_FILE)
```

Comment:
Can't this just be named EXPORT\_FILE\_FLAG or something along those lines? LDFLAGS\_NO\_GTEST seems a bit strange

Reply:
The thought was to clearly communicate why this must not be included in the gtest LDFLAGS; otherwise they must be exactly identical.